tachyon/math/polynomials/multivariate/multilinear_extension.h

```diff
@@ -166,7 +168,33 @@ H AbslHashValue(H h, const MultilinearExtension<Evaluations>& mle) {
   return H::combine(std::move(h), mle.evaluations());
 }
 
-}  // namespace tachyon::math
+}  // namespace math
+namespace base {
+
+template <typename Evaluations>
+class Copyable<math::MultilinearExtension<Evaluations>> {
+ public:
+  static bool WriteTo(const math::MultilinearExtension<Evaluations>& poly,
+                      Buffer* buffer) {
+    return buffer->Write(poly.evaluations());
+  }
+
+  static bool ReadFrom(const ReadOnlyBuffer& buffer,
+                       math::MultilinearExtension<Evaluations>* poly) {
+    Evaluations evaluations;
+    if (!buffer.Read(&evaluations)) return false;
+    *poly = math::MultilinearExtension<Evaluations>(std::move(evaluations));
+    return true;
+  }
+
+  static size_t EstimateSize(
+      const math::MultilinearExtension<Evaluations>& poly) {
```

Comment:
```suggestion
      const math::MultilinearExtension<Evaluations>& mle) {
```
How about like this for consistency?